arcaflow_plugin_sysbench/sysbench_schema.py

```diff
@@ -223,6 +249,116 @@ class SysbenchMemoryInputParams(CommonInputParameters):
     ] = SeqRnd.SEQ
 
 
+@dataclass
+class SysbenchIoInputParams(CommonInputParameters):
+    """
+    This is the data structure for the
+    input parameters of Sysbench CPU benchmark.
```

Comment:
Extremely minor nit -- Should be "IO benchmark" not "CPU benchmark"

Reply:
fixed and pushed.